core/services/job/job_pipeline_orm_integration_test.go

```diff
@@ -46,29 +43,29 @@ func TestPipelineORM_Integration(t *testing.T) {
 		BaseTask: pipeline.NewBaseTask("answer2", nil, 1, 0),
 	}
 	ds1_multiply := &pipeline.MultiplyTask{
-		Times:    decimal.NewFromFloat(1.23),
+		Times:    "1.23",
 		BaseTask: pipeline.NewBaseTask("ds1_multiply", answer1, 0, 0),
 	}
 	ds1_parse := &pipeline.JSONParseTask{
-		Path:     []string{"one", "two"},
+		Path:     "one,two",
 		BaseTask: pipeline.NewBaseTask("ds1_parse", ds1_multiply, 0, 0),
 	}
 	ds1 := &pipeline.BridgeTask{
 		Name:     "voter_turnout",
 		BaseTask: pipeline.NewBaseTask("ds1", ds1_parse, 0, 0),
 	}
 	ds2_multiply := &pipeline.MultiplyTask{
-		Times:    decimal.NewFromFloat(4.56),
+		Times:    "4.56",
 		BaseTask: pipeline.NewBaseTask("ds2_multiply", answer1, 0, 0),
 	}
 	ds2_parse := &pipeline.JSONParseTask{
-		Path:     []string{"three", "four"},
+		Path:     "three,four",
 		BaseTask: pipeline.NewBaseTask("ds2_parse", ds2_multiply, 0, 0),
 	}
 	ds2 := &pipeline.HTTPTask{
-		URL:         models.WebURL(*u),
+		URL:         "https://chain.link/voter_turnout/USA-2020",
 		Method:      "GET",
-		RequestData: pipeline.HttpRequestData{"hi": "hello"},
+		RequestData: `{"hi": "hello"}`,
```

Comment:
Interesting... it was necessary to make everything a string so the variables can be parsed later?

Reply:
The downside of this is that we lose type-safety at compile time of our tasks :(

Reply:
No worries, either @archseer or I will be doing a followup that solves this